app/code/Magento/Theme/Model/Design/Backend/File.php

```diff
@@ -247,7 +247,7 @@ private function getMime()
      */
     private function getRelativeMediaPath(string $path): string
     {
-        return preg_replace('/\/(pub\/)?media\//', '', $path);
+        return preg_replace('/\/(pub\/)?media\//', '', strstr($path, '/pub/media/'));
```

Comment:
Looks like we're getting url here.

It will work for cases `https://magento2.dev/mymagento/pub/media/...` and  `https://magento2.dev/pub/media/...`.
But we might have a case when URL could be `https://magento2.dev/mymagento/media/...` or `https://magento2.dev/media/...`, in this case it will not work. It will cause regression.

Reply:
@ihor-sviziev, okay I'm checking for the required solution.